azext_edge/edge/commands_asset_endpoint_profiles.py

```diff
@@ -58,28 +79,42 @@ def delete_asset_endpoint_profile(
     asset_endpoint_profile_name: str,
     resource_group_name: str,
 ) -> dict:
-    aep_provider = AssetEndpointProfileProvider(cmd)
-    return aep_provider.delete(
+    return AssetEndpointProfiles(cmd).delete(
         asset_endpoint_profile_name,
-        resource_group_name=resource_group_name,
-        check_cluster_connectivity=True
+        resource_group_name=resource_group_name
+    )
+
+
+# TODO: add in once GA
+def list_asset_endpoint_profiles(
+    cmd,
+    resource_group_name: str,
+) -> dict:
+    return AssetEndpointProfiles(cmd).list(
+        resource_group_name=resource_group_name
     )
 
 
 def query_asset_endpoint_profiles(
     cmd,
-    additional_configuration: Optional[str] = None,
+    asset_endpoint_profile_name: Optional[str] = None,
     auth_mode: Optional[str] = None,
-    custom_location_name: Optional[str] = None,
+    custom_query: Optional[str] = None,
+    endpoint_profile_type: Optional[str] = None,
+    instance_name: Optional[str] = None,
+    instance_resource_group: Optional[str] = None,
     location: Optional[str] = None,
     resource_group_name: Optional[str] = None,
     target_address: Optional[str] = None,
-) -> dict:
-    aep_provider = AssetEndpointProfileProvider(cmd)
-    return aep_provider.query(
-        additional_configuration=additional_configuration,
+) -> List[dict]:
+    return AssetEndpointProfiles(cmd).query_asset_endpoint_profiles(
+        asset_endpoint_profile_name=asset_endpoint_profile_name,
         auth_mode=auth_mode,
-        custom_location_name=custom_location_name,
+        custom_query=custom_query,
+        discovered=False,
```

Comment:
this is always False and not able to modify by user?

Reply:
This will be changeable once I put in discovery support.